Xamarin.Essentials/Barometer/Barometer.android.cs

```diff
@@ -40,6 +40,11 @@ void ISensorEventListener.OnAccuracyChanged(Sensor sensor, [GeneratedEnum] Senso
         }
 
         void ISensorEventListener.OnSensorChanged(SensorEvent e)
-            => Barometer.OnChanged(new BarometerData(e.Values[0]));
+        {
+            if ((e?.Values?.Count ?? 0) == 0)
```

Comment:
I'm just paranoid but... 

```suggestion
            if ((e?.Values?.Count ?? 0) <= 0)
```